src/js/prediction.js

```diff
@@ -120,6 +130,8 @@ class Click {
         prediction: response.prediction,
       };
 
+      // console.log('predic',trial_data.prediction)
+
```

Comment:
Oh, here's another one.

Reply:
Nevermind, fixed in https://github.com/sbruinsma1/adLearning_web_HC/pull/7/commits/5f7c9b02d63940a3fc160dff53bb94c4ff41965e.